docs/notifications/03-showrunners-scaffold/03-Examples/07-Tutorial-Notification-Settings-With-Showrunners.mdx

```diff
@@ -0,0 +1,118 @@
+---
+id: docs-notifications-settings-with-showrunners-example
```

Comment:
Frontmatter should follow the subfolder nomenclature, will make the edit and merge